### PR TITLE
 myDividends24 export

### PR DESCRIPTION
**Pull request by Hasakirii**
- Added new Files (MyDividends24Uploader, Handler and PreferencesPage)
- Added with resource-bundle editor all necessary message/bundle properties
- Added Upload Section to Menu
- Added 2 Icons with 2 sizes
- Added myDividends24 information to WelcomePart

### CODE REVIEW

File: name.abuchen.portfolio/src/name/abuchen/portfolio/online/impl/MyDividends24Uploader.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
+package name.abuchen.portfolio.online.impl;
```
Comment (Nirus2000) [inline name.abuchen.portfolio/src/name/abuchen/portfolio/online/impl/MyDividends24Uploader.java:1]: Maybe this is a better solution:
[MyDividends24Uploader.java.txt](https://github.com/buchen/portfolio/files/11238578/MyDividends24Uploader.java.txt)

If you have time, you can check this out.

Comment (Hasakirii) [reply name.abuchen.portfolio/src/name/abuchen/portfolio/online/impl/MyDividends24Uploader.java:1]: I tested it, with your implementation, the upload doesn't work. 
Is this the only problem why the Build didn't succeed?

**Comment by buchen** [comment]
Hi @Hasakirii, I looked over the code. I have some comments but overall it looks okay. But before adding such a feature, let's understand what data you can provide to PP in return. I see this as a give and take. Feel free to contact me on portfolio dot performance dot help at gmail dot com.

**Comment by Hasakirii** [comment]
> Hi @Hasakirii, I looked over the code. I have some comments but overall it looks okay. But before adding such a feature, let's understand what data you can provide to PP in return. I see this as a give and take. Feel free to contact me on portfolio dot performance dot help at gmail dot com.

I wrote you an email with all necessary information, thank you.

**Pull request closed without merging**